web/src/features/projects/server/projectsRouter.ts

```diff
@@ -160,13 +193,15 @@ export const projectsRouter = createTRPCRouter({
       const project = await ctx.prisma.project.findUnique({
         where: {
           id: input.projectId,
+          deletedAt: null,
```

Comment:
did you check all cases where we fetch the project from db? We need to add this condition everywhere.

Reply:
also we need to make everyone from the team aware

Reply:
Yes, I updated all calls where we use prisma.project and did not find any raw queries that reference the projects table.

I'll send an update via #engineering after merging this.